test/e2e/e2e.go

```diff
@@ -76,6 +86,12 @@ var _ = Describe("validation", func() {
 			err := testclient.Client.Get(context.Background(), goclient.ObjectKey{Name: MetallbOperatorCRDName}, crd)
 			Expect(err).ToNot(HaveOccurred())
 		})
+
+		It("should have the MetalLB AddressPool CRD available in the cluster", func() {
```

Comment:
maybe we should consider using a function for CRDs instead of creating an It for each one (could be useful if more CRDs are added). thoughts?

Reply:
yeah not sure if there is better ways to do so at least in this release there are no more CR but definitely more down the road

Reply:
it is possible with DescribeTable (afaik), and seems simple to implement a func that checks if a given CRD is present (I could be wrong here)

Reply:
Let's not overengineer when not needed.
If there are no other CRDs, let's not add tables. We can still switch back when we introduce new ones.